lutris/runners/runner.py

```diff
@@ -18,10 +18,14 @@
 from lutris.util.linux import LINUX_SYSTEM
 from lutris.util.log import logger
 
+if TYPE_CHECKING:
```

Comment:
Oh, I wish I had known about TYPE_CHECKING! I'll definitely be able to use that.